blockonomics-woocommerce.php

```diff
@@ -40,29 +40,32 @@
     exit; // Exit if accessed directly
 }
 
-include_once( ABSPATH . 'wp-admin/includes/plugin.php' );
+require_once ABSPATH . 'wp-admin/includes/plugin.php';
 
-if ( is_plugin_active( 'woocommerce/woocommerce.php') || class_exists( 'WooCommerce' )) {
+if (is_plugin_active('woocommerce/woocommerce.php') || class_exists('WooCommerce')) {
     function blockonomics_woocommerce_init()
     {
         if (!class_exists('WC_Payment_Gateway')) {
             return;
         }
+
```

Comment:
Is this the file where user is suppose to add his custom css ? We need to create this file with comments as shown here https://www.elegantthemes.com/blog/tips-tricks/how-to-safely-add-custom-css-styles-to-plugins